components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -146,6 +162,31 @@ service ReportingService {
 	};
 }
 
+message ControlItemRequest {
+	string text = 1;
+	int32 size = 2;
+	repeated ListFilter filters = 3;
+}
+
+message StringTag {
+	string key = 1;
+	repeated string values = 2;
+}
+
+message ControlItem {
+	string id =1;
+	string title=2;
+	ProfileMin profile = 3;
+	float impact = 4;
+	google.protobuf.Timestamp end_time = 5;
+	ControlSummary control_summary = 6;
+	repeated StringTag string_tags = 7;
```

Comment:
Are sting_tags required for this API? I'm asking because this is the internal representation that we have of the control tags that we support for filtering(`null` value or single string or array of strings value). Any other values will not be stored as string_tags in elastic search.